manifests/s/Sinew/Enpass/6.11.0/Sinew.Enpass.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Sinew.Enpass
+PackageVersion: 6.11.0
+InstallerType: burn
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- enpass
+- enpassauth
+- enpasscard
+- enpassshare
+- enpassstart
+FileExtensions:
+- enpassbackup
+- enpasscard
+- enpassdb
+ProductCode: '{4bf661d1-0360-4421-9e10-ad36eadead0e}'
+AppsAndFeaturesEntries:
+- ProductCode: '{4bf661d1-0360-4421-9e10-ad36eadead0e}'
+  UpgradeCode: '{0BB552C5-B5B4-406D-9F42-9FA62580F18A}'
+Installers:
+- Architecture: x86
+  InstallerUrl: https://dl.enpass.io/stable/windows/setup/6.10.1.1661/Enpass-setup.exe

```

Comment:
Hi @PckgrBot,

This `InstallerUrl` points at the previous version of the package. Updating to this PR's version results in an erroring URL:

```suggestion
  InstallerUrl: https://dl.enpass.io/stable/windows/setup/6.11.0/Enpass-setup.exe
```
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/c59053d8-09de-485f-9623-d0b652f92064)

Any suggestions?